tests/serializers.py

```diff
@@ -59,11 +59,15 @@ class Meta:
             'cats', 'friendly_cats', 'bad_cats'
         )
 
+    def filter_queryset(self, query):
```

Comment:
Don't we usually define functions after fields in serializers?  As a style thing, that is.

Reply:
Yep, you're right. Moved.